containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
+    && apk add --no-cache build-base git curl tzdata zlib-dev nodejs yarn libc6-compat sqlite-dev mariadb-dev libpq-dev
+
+ENV APP_ROOT=/opt/PasswordPusher PATH=${APP_ROOT}:${PATH} HOME=${APP_ROOT}
+
+RUN mkdir -p ${APP_ROOT}
+COPY ./ ${APP_ROOT}/
+
+WORKDIR ${APP_ROOT}
+
+RUN gem install bundler
+
+ENV RACK_ENV=production RAILS_ENV=production
+
+RUN bundle config set without 'development private test' \
+    && bundle config set deployment 'true' \
+    && bundle config set with 'sqlite mysql postgres' \
```

Comment:
Can be changed out of variables:
```dockerfile
ENV BUNDLE_WITH=sqlite:mysql:postgres
ENV BUNDLE_WITHOUT=development:test:private
ENV BUNDLE_DEPLOYMENT=true
```

This way the `bundle install` command is static.

My two cents that is.

Reply:
I made these variables ARG not ENV.
They can be used as parameters while building then.